### ISSUE TITLE
ValueError: zero-size array to reduction operation maximum which has no identity

### ISSUE THREAD

**Issue by vikash130795** [VikramShenoy97/Music-Recommendation-Using-Deep-Learning#1]
Hi,

 Whenever, Am running train.py. I'm getting this error:

ValueError: zero-size array to reduction operation maximum which has no identity

**Comment by KartikayKumar02** [comment]
Yes even I am having the same issue. I tried rollbacking my scikitlearn version as well to .19 to see if the issue arose in the later versions but it still existed. My guess is that the data isnt getting one hot encoded.

**Comment by Antoninovenuto95** [comment]
Any update?
